docs/reference/query-dsl/script-score-query.asciidoc

```diff
@@ -4,18 +4,19 @@
 <titleabbrev>Script score</titleabbrev>
 ++++
 
-The `script_score` allows you to modify the score of documents that are
-retrieved by a query. This can be useful if, for example, a score
-function is computationally expensive and it is sufficient to compute
-the score on a filtered set of documents.
+Uses a <<modules-scripting,script>> to modify the <<relevance-scores,relevance
+scores>> of another query.
 
-To use `script_score`, you have to define a query and a script -
-a function to be used to compute a new score for each document returned
-by the query. For more information on scripting see
-<<modules-scripting, scripting documentation>>.
+The `script_score` query is useful if, for example, a scoring function is
+expensive and you only need to calculate the relevance score of a filtered set
```

Comment:
the relevance score -> score
Here a user can compute any score they like, it could be completely irrelevant to the IR relevance.

Reply:
Fixed with a31f4a8ae60f8bbae76f211f53e1e88ee1a49541.